codelabs/codelab-02/codelab2README.md

```diff
@@ -0,0 +1,501 @@
+#Codelab 2: AWS + S3
+
+### Due Date
+
+This codelab is due on *Thursday, February 8th at 11:59:59PM*.
```

Comment:
Should we give them two weeks? And mention that a part B will be added on CloudFront

Reply:
And that both parts will be due then.

Maybe add a note that this codelab looks long, but a lot of it is just setup + casual walkthrough